ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2948,7 +2948,7 @@ tests:
   steps:
     cluster_profile: vsphere
     env:
-      E2E_RUN_TAGS: '@vsphere-upi and @network-ovnkubernetes and not @fips'
+      E2E_RUN_TAGS: '@baremetal-upi and @network-ovnkubernetes and not @fips'
```

Comment:
so there will be no `vsphere-upi `anymore ?

Reply:
I think that should not work, because there is no such profile, I add a comment here, @miyadav PTAL, does it look better?

Reply:
sorry @huali9 dont think comment is allowed 
```
`ERROR: This check enforces that CI Operator configuration YAML files are generated
ERROR: correctly. We have automation in place that updates these configs and
ERROR: new changes to these configurations should be followed with a re-generation.
ERROR: Run the following command to re-generate the CI Operator configurations:
ERROR: $ make ci-operator-config
ERROR: The following errors were found:
diff -Naupr .//ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml /tmp/tmp.juUrdbzqdR/ci-operator/config/openshift/openshift-tests-private/openshift-.
.
.

```

Hence I am ok , with it , if everyone is ok as well . 
LGTM

Reply:
you might need to remove comment.

Reply:
Yeah, seems that, changed back, thanks @miyadav